app/routes/adapter-files.js

```diff
@@ -12,13 +12,23 @@ const Express = require('express');
 const MetaScript = require('metascript');
 
 const Router = Express.Router();
+const partnerName = require('../../bin/index');
 
 // =============================================================================
 // MAIN ////////////////////////////////////////////////////////////////////////
 // =============================================================================
 
 const cwd = Process.cwd();
-const fileNames = Fs.readdirSync(cwd);
+var dir = cwd;
```

Comment:
Definitely sure your linter isn't working now. For all the new JS stuff we do, we're strictly using `let` and not `var`.